commit 1ceb47f042257dbd3eefe1b74ce583d6049dbb0e
Author: JamesH65 <JamesH65@users.noreply.github.com>
Date:   Mon Feb 6 15:24:47 2017 +0000

    gpio_mem: Remove unnecessary dev_info output (#1830)
    
    The open function was spamming syslog every time
    called, so have removed call completely.

diff --git a/drivers/char/broadcom/bcm2835-gpiomem.c b/drivers/char/broadcom/bcm2835-gpiomem.c
index 911f5b7393ed..f5e7f1ba8fb6 100644
--- a/drivers/char/broadcom/bcm2835-gpiomem.c
+++ b/drivers/char/broadcom/bcm2835-gpiomem.c
@@ -76,8 +76,6 @@ static int bcm2835_gpiomem_open(struct inode *inode, struct file *file)
 	int dev = iminor(inode);
 	int ret = 0;
 
-	dev_info(inst->dev, "gpiomem device opened.");
-
 	if (dev != DEVICE_MINOR) {
 		dev_err(inst->dev, "Unknown minor device: %d", dev);
 		ret = -ENXIO;
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1139_1ceb47f042257dbd3eefe1b74ce583d6049dbb0e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1139_1ceb47f042257dbd3eefe1b74ce583d6049dbb0e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
